src/components/Login/Login.jsx

```diff
@@ -2,7 +2,7 @@ import { useEffect } from "react";
 import { useHistory } from "react-router-dom";
 import Sawo from "sawo";
 import logo from "../../assets/login/logo.svg";
-import "./Login.css";
+import "./Login.module.css";

```

Comment:
Same here also. Check out the previous review.

Reply:
Ok